packages/frontend/src/retro/components/TimePicker.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { FlexBox } from "../../common/components/FlexBox";
+import { TextInput } from "../../common/components/TextInput";
+import IncrementTimerButton from "./buttons/IncrementTimerButton";
+interface TimePickerProps {
+  minutes: string;
+  seconds: string;
+  isEditable: boolean;
```

Comment:
I'd also use the classic known term `isDisabled` or `disabled` here and invert the conditions